docs/development/l10n.rst

```diff
@@ -293,6 +293,32 @@ like Google, Paypal and Twitter now offer two-factor authentication.
 
 This definition was copied from `the EFF glossary <https://ssd.eff.org/en/glossary/two-factor-authentication>`__
 
+Weblate glossary
+----------------
+
+For each string to be translated, Weblate shows a glossary of terms
+and their translation to help unify their translations. For instance
+when translating `Please wait for a new two-factor token before
+logging in again`, Weblate notices the word `two-factor` is found in
+the glossary and displays the translation in the `glossary` to the
+right.
+
+|Weblate glossary show page screenshot|
+
+Before translating strings, it is recommended to add all term in the
```

Comment:
`all term` -> `all terms`

Reply:
done. Funny that was the same mistake in another PR. The **s** does not suit **term** apparently ;-)